Microsoft.Identity.Web/Client/MsalUiRequiredExceptionFilterAttribute.cs

```diff
@@ -37,6 +46,20 @@ public override void OnException(ExceptionContext context)
             {
                 if (CanBeSolvedByReSignInUser(msalUiRequiredException))
                 {
+                    if (configuration == null && !string.IsNullOrWhiteSpace(ScopeKeySection))
```

Comment:
@TiagoBrenck 
I believe that you can get hold of the configuration via dependency injection (and this would work with whatever data sources were proposed in the main). Here you are redoing something that was already done elsewhere

Reply:
According to the documentation, DI cannot be applied on `ExceptionFilterAttribute`: https://docs.microsoft.com/en-us/aspnet/core/mvc/controllers/filters?view=aspnetcore-2.2#dependency-injection

Reply:
yes, I know, @TiagoBrenck, which is why I said you might have to delegate to another singleton object created by DI:
MsalUiRequiredExceptionFilterAttribute uses a singleton class which imports by DI the configuration.